### ISSUE TITLE
Model training results are not reproducible

### ISSUE THREAD

**Issue by ADamon912** [SindiLab/ACTINN-PyTorch#3]
In your model, I fixed the random seed in the global, and also fixed the random seed when initializing the weight, and fixed the order of DataLoader data reading according to the pytorch official website document, why the results of each training are different, the code is as follows, I hope to answer the doubts, thank you very much

```
#global seed 
seed = 3
torch.manual_seed(seed)
```
```
def init_weights(model, seed=3):
    torch.manual_seed(seed)
    if isinstance(model, torch.nn.Linear):
        torch.nn.init.xavier_uniform_(model.weight)
        model.bias.data.fill_(0.01)
```
```
#DataLoader

def seed_worker(worker_id):
    worker_seed = torch.initial_seed() % 2**32
    numpy.random.seed(worker_seed)
    random.seed(worker_seed)
g_cuda = torch.Generator(device='cuda')
g.manual_seed(3)
    
train_data_loader = DataLoader(data_and_labels, batch_size=batchSize, num_workers=workers, 
        worker_init_fn=seed_worker, generator=g)
```

**Comment by dr-aheydari** [comment]
Hi @ADamon912,

I'm not sure which results you are referring to, given that we do not have a specific paper for this package alone (and do not report results in the repo here). Please be more specific about which of our datasets you are using, which version of `torch` you are running our models on, etc.

If you are referring to results of training ACTIVA (a different model) with ACTINN, please open an issue in the [ACTIVA repo](https://github.com/SindiLab/ACTIVA).

I'll be closing the issue for now, but please feel free to open another issue in the appropriate repo, or to reopen the issue here providing which of our experiments you are trying to reproduce :)
